### DISCUSSION TITLE
How to build from source on Mac?

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#312] [Q&A]
I have a mac mini that is running Catalina 10.15.7. It has a 2.5Ghz Intel I5 processor with 16GB ram.

How can I build SDR++ from source code? The documentation says "TBD" unless I missed something.

thank you

**Comment by AlexandreRouma** [comment]
exact same way as you would build on linux

**Reply by pmaine** [reply]
I installed homebrew on my mac. I next ran the command: brew install glew. Brew successfully installed glew but when I try a cmake .. from the build directory it can find glew. If you have an alternative method I am open to using your method.

**Reply by pmaine** [reply]
it cannot find glew

**Reply by pmaine** [reply]
Can I please get some help on the mac install? I have built on a Pi4 and Ubuntu successfully. The usual steps do not work.
